src/networks.json

```diff
@@ -1144,6 +1144,22 @@
     "start": 19454413,
     "logo": "ipfs://bafkreihh4y35xgshx7wbf7a2xj35do55f7lj4nhxqxtt5nqoohoojdvkki"
   },
+  "2152": {
+    "key": "2152",
+    "name": "Findora Mainnet",
+    "shortName": "Findora",
+    "chainId": 2152,
+    "network": "mainnet",
+    "multicall": "0x319E6CCaA89C3dCe65946E759a32a09531E6b72f",
+    "rpc": [
+      "http://archive.prod.findora.org:8545/"
```

Comment:
@ws4charlie This node doesn't look be working for me, It throw `Error: could not detect network` maybe it will not work on front end because of CORS?

Reply:
@ChaituVR Thanks for your report. Let me check the configuration with devops team and quickly post my update in here by tomorrow.

Reply:
@ChaituVR The node `http://archive.prod.findora.org:8545` is working now. Can you please give it a try and feel free to post your blockers in here?

Reply:
@ws4charlie I get same error 😢 I am trying it from chrome browser

Reply:
@ChaituVR Here is my Metamask network configuration and it works very well. What command returned you `Error: could not detect network`?  Can you try `curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}' http://archive.prod.findora.org:8545/` and see if you can get the block number? If you can, that means the error might come from somewhere else.

![image](https://user-images.githubusercontent.com/34498985/210896991-e9ee0c2b-32ac-4a32-995e-335085a280cb.png)

Reply:
@ChaituVR My telegram handle is `@charliecc` and my Discord handle is `charliecc#0988`. Please feel free to connect. I'm glad to join any debugging session/meeting (google meeting, zoom, etc.) if you would like to share your screen. 

The url `http://archive.prod.findora.org:8545` is a very standard implementation of web3 APIs. I was wondering if above error came from websocket which is offered on `http://archive.prod.findora.org:8546`.

Reply:
![Untitled](https://user-images.githubusercontent.com/15967809/211501346-776aebf7-f44b-442a-9080-835db8e49372.png)

This is the error I am getting, I think this is because of the `http`, ether.js automatically converts `http` to `https` and this URL doesn't work with `https`.
(the curl command works but not in the browser using ethers.js)

https://rpc-mainnet.findora.org This node works fine, but it is not a full archive, only works with a certain range of blocks.

Reply:
@ChaituVR Thanks for your investigation and debugging. We'll fix the http issue soon and get back to you soon.

Reply:
@ChaituVR We switched to `https` and please give it a try whenever you get the chance. Thank you very much.

Reply:
Hi @ChaituVR , any update/blocker on integration?